# Holds a graph 
# - nodes: list of nodes, ie skel_disk_graph_roadmap/DiskGraphNode
# - nodesIdsMapping: list of nodes indices in order
# - edges: list of skel_disk_graph_roadmap/DiskGraphEdge
skeleton_disk_graph_roadmap/DiskGraphNode[] nodes
int32[] nodesIdsMapping
skeleton_disk_graph_roadmap/DiskGraphEdge[] edges
